### PR TITLE
product req spec should be a website

### PR DESCRIPTION
**Pull request by Codebreake**
I changed the "web" type to "url" and the error message come automatically when user clicks create

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -42,4 +42,4 @@
     ],
     "development": []
   }
-}
+}
```
Comment (powermukisa) [inline package.json]: new line at EOF. why was the extra line deleted?

File: src/components/features/createfeature/CreateFeature.js [hunk 2]
```diff
@@ -121,7 +121,7 @@ const CreateFeature = (props) => {
   return (
       <div id='upload'>
         <h3>Create a Feature</h3>
-        <form onSubmit={handleCreate} style={{marginTop: '25px'}}>
+        <form onSubmit={ handleCreate } style={{marginTop: '25px'}}>
```
Comment (powermukisa) [inline src/components/features/createfeature/CreateFeature.js]: what's reason for this change?

File: src/components/features/createfeature/CreateFeature.js [hunk 3]
```diff
@@ -176,7 +176,7 @@ const CreateFeature = (props) => {
                 margin="dense"
                 id="productSpec"
                 label="Product Requirement Spec"
-                type="web"
+                type="url"
```
Comment (powermukisa) [inline src/components/features/createfeature/CreateFeature.js]: awesome. could you do the same for the tech spec too? also look through the app and see where this is required. There are lots of places this needs to be fixed

File: src/components/features/createfeature/CreateFeature.js (hunk 2 above)
Comment (Codebreake) [reply src/components/features/createfeature/CreateFeature.js]: This wasnt suposed to change , i had added some code and deleted it later. I think thats how the space came about

**Pull request closed without merging**